project/api/tests.py

```diff
@@ -407,24 +366,75 @@ def test_api_can_not_query_territories_date(self):
         response = self.client.get(url, format="json")
         self.assertEqual(response.status_code, status.HTTP_200_OK)
         self.assertTrue(not response.data)
-
-    def test_api_can_query_nation(self):
+    
+    def test_api_can_query_territories_exclude(self):
```

Comment:
This is the only new testcase

Reply:
I'd love to see another test case (or replace this one), by having at least 2 territories, excluding one with `exclude_ids`, and the api returns the other one.

Reply:
I think I'll rewrite the test file in this PR when I get the time:
* Use factories instead of objects.create
* Fix some of the assertEqual constants